indexeddb-files/tabs/indexeddb-filehandle.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Create temporary files</title>
+    <meta charset="utf-8">
+    <style>
+     .overflow-scroll {
+       max-height: 200px;
+       overflow: auto;
+     }
+    </style>
+  </head>
+  <body>
+    <h1>Create mutable files using the MutableFile API, and then save and retrieve them using IndexedDB API</h1>
```

Comment:
This is quite long! Could we make the H1 "Create and save temporary files" and then have a P underneath giving the details, like "Create a temporary file, give it some data and a name, then save it using indexedDB".